src/app/forms/utils/enum-map.ts

```diff
@@ -0,0 +1,5 @@
+export default function getOptionsFromObject(object: object): object[] {
+  return Object.values(object).map((value) => {
+      return {value: value, label: value}
+  })
+}
```

Comment:
- There is a `MultiOptions` interface better suited to the return type of the function
- Can be a bit more concise by wrapping the object in `()` so you can return on the same line
- You can just but `value` in your object because it has the same name as your property

```suggestion
import { MultiOptions } from '@forms/models/options.model';

export default function getOptionsFromObject(object: object): MultiOptions {
  return Object.values(object).map(value => ({value, label: value}))
}
```

Reply:
eeeerrrm, 
Nathan just helped me reformat that to:

export default function getOptionsFromObject(object: object): FormNodeOption<string | number | boolean>[] {
  return Object.values(object).map(value => {
    return { value: value, label: value };
  });
}

Now IDK what to do

Reply:
As far as I can tell the return types we have given you are identical, one is just pre-created for this, I'd maybe move it over to use that type rather than the declared one I gave. In all honestly, I just didn't know this existed

Reply:
Ok, I used the new one, and it removed your tick.. QQ